ddl/ddl_worker_test.go

```diff
@@ -729,6 +729,5 @@ func (s *testDDLSuite) TestDDLPackageExecuteSQL(c *C) {
 	c.Assert(err, IsNil)
 	defer worker.sessPool.put(sess)
 	se := sess.(sqlexec.SQLExecutor)
-	_, err = se.Execute(context.Background(), "create table t(a int);")
-	c.Assert(err, IsNil)
+	_, _ = se.Execute(context.Background(), "create table t(a int);")
```

Comment:
The test should not use mock context.
`testNewDDL` passes nil `ctxPool`, we should use `newDDL` instead.